lstchain/io/event_selection.py

```diff
@@ -186,16 +183,33 @@ def apply_global_theta_cut(self, data):
         return data[data["theta"].to_value(u.deg) < self.global_theta_cut]
 
     def energy_dependent_theta_cuts(
-        self, data, energy_bins, smoothing=None,
+        self,
+        data,
+        energy_bins,
+        use_same_disp_sign=True,
+        smoothing=None,
     ):
         """
-        Evaluating an optimized energy-dependent theta cuts, in a given
-        data, with provided reco energy bins, and other parameters to
-        pass to the pyirf.cuts.calculate_percentile_cut function.
+        Evaluating an optimized energy-dependent theta cuts, in a given MC data,
```

Comment:
Please clarify, "optimized" is a bit misleading. We simply evaluate the cut which has a given efficiency on a given sample of MC gammas (namely, those with correct disp_sign).